cmd/umserver/graceful_shutdown.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"log"
+	"net/http"
+	"sync"
+	"time"
+)
+
+func gracefulShutdown(timeout time.Duration, wg *sync.WaitGroup, srv *http.Server) error {
```

Comment:
How would you close connections to postgres and consul?

Reply:
Thanks. I fixed it.